backend/app/Console/Commands/ImportLegacyDataCommand.php

```
@@ -0,0 +1,454 @@
+            }
+        }
+
+        try {
+            DB::transaction(function () use ($legacy, $chunk): void {
+                $this->importUsers($legacy, $chunk);
+                $this->importArticles($legacy, $chunk);
+                $this->importTags($legacy, $chunk);
+                $this->importBookMarks($legacy, $chunk);
+                $this->importArticleTags($legacy, $chunk);
+                $this->importBookMarkTags($legacy, $chunk);
+            });
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'handle'. (undefined)

(StaticAccess)